Client/src/Models/Datasets/ITemporaryModel.ts

```diff
@@ -0,0 +1,41 @@
+export interface IDataPointModel {
```

Comment:
what is this modal for, also the interface name is IDataPointModel but the file's name is ITemporaryModel

Reply:
model*

Reply:
That was something I was using before, but it's not necessary now. I'll delete it.

Reply:
Reminder to remove it as well from the imports